### PR TITLE
iOS table view: support multiple sections

### PR DESCRIPTION
**Pull request by adly-holler**
🌀

### CODE REVIEW

**Comment by adly-holler** [comment]
Before this gets merged, we should apply the parent element fix from #68

File: Few-iOS/TableView.swift [hunk 1]
```diff
@@ -12,6 +12,26 @@ private var ElementKey = "ElementKey"
...
+	private var realizedElement: RealizedElement?
+	
+	func updateWithElement(element: Element) {
+		if let realizedElement = realizedElement {
+			if element.canDiff(realizedElement.element) {
+				element.applyDiff(realizedElement.element, realizedSelf: realizedElement)
+			} else {
+				realizedElement.remove()
+				
+				let parent = RealizedElement(element: Element(), view: contentView, parent: nil)
```
Comment (joshaber) [inline Few-iOS/TableView.swift]: As in #68, I think we need to use a consistent parent.

**Comment by joshaber** [comment]
This is awesome :cookie: Does this supersede #50?

**Comment by adly-holler** [comment]
Thanks man! No it doesn't – #50 will give us the non-pinned global header, this gives us the pinned section headers/footers.

**Comment by adly-holler** [comment]
@joshaber submitted for the approval of the midnight society

**Comment by adly-holler** [comment]
Dora we meet again, at last. The circle is now complete. When I left you I was but the learner; now I am the master.

**Pull request merged**